internal/store/elasticsearch/es.go

```diff
@@ -65,6 +60,40 @@ type searchResponse struct {
 	} `json:"aggregations"`
 }
 
+type groupResponse struct {
+	Aggregations struct {
+		CompositeAggregations struct {
+			Buckets []bucket `json:"buckets"`
+		} `json:"composite-group"`
+	} `json:"aggregations"`
+}
+
+type bucket struct {
```

Comment:
Can we remove this and use the type `aggregationBucket` instead? Both are identical.

Reply:
done